source/git-util.js

```diff
@@ -8,6 +8,14 @@ exports.latestTag = async () => {
 	return stdout;
 };
 
+exports.newFilesSinceLastRelease = async () => {
+	const {stdout} = await execa('git', ['diff', '--stat', '--diff-filter=A', await this.latestTag(), 'HEAD']);
+	const result = stdout.trim().split('\n').map(row => row.slice(0, row.indexOf('|')).trim());
```

Comment:
```suggestion
	const result = stdout.trim().split('\n').slice(0, -1).map(row => row.slice(0, row.indexOf('|')).trim());
```

This makes line 15 unnecessary and simplifies the code.